### ISSUE TITLE
can't pause music on wired headphones

### ISSUE THREAD

**Issue by teacupmaps** [koreader/koreader#11655]
* KOReader version: 2024.03.1
* Device:Galaxy S9

#### Issue
while koreader is open, the pause button doesn't work
#### Steps to reproduce
1. start playing audio in any app, for example, newpipe
2. open koreader

**Comment by pazos** [comment]
Hi, verbose logs please.

We already forward a bunch of media keys to the system. Maybe we miss yet another one?

https://github.com/koreader/koreader-base/blob/master/ffi/input_android.lua#L177-L185

**Comment by teacupmaps** [comment]
i don't have a crash.log file

**Comment by Frenzie** [comment]
I believe you can create one somewhere under help. (Top right menu.)

**Comment by pazos** [comment]
@teacupmaps: follow https://github.com/koreader/koreader/issues/5761#issuecomment-573345775

Whatever it is, it seems out of spec: https://source.android.com/docs/core/interaction/accessories/headset/jack-headset-spec

**Comment by teacupmaps** [comment]
i pressed the pause button 16 times and clicked report issue in help:

```
~/storage/shared $ grep -o "key event => code: .*, value: 1" koreader/crash.log | uniq
key event => code: 79 (nil), value: 1
~/storage/shared $ grep -o "key event => code: .*, value: 1" koreader/crash.log | wc -l
16
~/storage/shared $
```

this is a "headset hook key": https://developer.android.com/reference/android/view/KeyEvent#KEYCODE_HEADSETHOOK

**Comment by pazos** [comment]
Fixed in #1774
